### PR TITLE
sippy should run golangci-lint

### PR DESCRIPTION
**Pull request by cheesesashimi**

### CODE REVIEW

**Comment by stbenjam** [comment]
I think you'll need to run `make jobs` too

**Comment by cheesesashimi** [comment]
This shouldn't be merged until https://github.com/openshift/sippy/pull/248 is, since that PR introduces the `make lint` target.

**Comment by stbenjam** [comment]
@cheesesashimi Looks like this needs a rebase

File: ci-operator/config/openshift/sippy/openshift-sippy-master.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ base_images:
     name: release
     namespace: openshift
     tag: golang-1.15
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.37.1
```
Comment (stbenjam) [inline ci-operator/config/openshift/sippy/openshift-sippy-master.yaml:9]: We moved to a custom build root (https://github.com/openshift/sippy/blob/master/Dockerfile.buildroot) for the eventual React frontend, I think you could add golangci-lint to it -- that way we can eventually run eslint from the same make target.

Comment (cheesesashimi) [reply ci-operator/config/openshift/sippy/openshift-sippy-master.yaml]: Thanks for letting me know. I've updated the config to add the `make lint` step with the assumption that it is in the buildroot image. I've opened a separate PR (https://github.com/openshift/sippy/pull/256) to add `golangci-lint` to the buildroot image, which should be merged before this one.

**Pull request merged**